config/crd/bases/atlas.mongodb.com_atlasdeployments.yaml

```diff
@@ -362,6 +362,35 @@ spec:
                     type: array
                   rootCertType:
                     type: string
+                  searchNodes:
+                    description: List of settings that configure the search nodes
+                      for your cluster
+                    items:
+                      properties:
+                        instanceSize:
+                          enum:
+                          - S20_HIGHCPU_NVME
+                          - S30_HIGHCPU_NVME
+                          - S40_HIGHCPU_NVME
+                          - S50_HIGHCPU_NVME
+                          - S60_HIGHCPU_NVME
+                          - S70_HIGHCPU_NVME
+                          - S80_HIGHCPU_NVME
+                          - S30_LOWCPU_NVME
+                          - S40_LOWCPU_NVME
+                          - S50_LOWCPU_NVME
+                          - S60_LOWCPU_NVME
+                          - S80_LOWCPU_NVME
+                          - S90_LOWCPU_NVME
+                          - S100_LOWCPU_NVME
+                          - S110_LOWCPU_NVME
+                          type: string
+                        nodeCount:
+                          maximum: 32
+                          minimum: 2
+                          type: integer
+                      type: object
+                    type: array
```

Comment:
if the API only allows one entry, we should limit it here as well.

Reply:
Done